framework/requirements/views/web/index.php

```diff
@@ -31,15 +31,15 @@ functionality may be not available in this case.
         </p>
 
         <h3>Conclusion</h3>
-        <?php if ($summary['errors'] > 0): ?>
+        <?php if ($summary['errors'] > 0) : ?>
```

Comment:
This seems to be correct: https://github.com/squizlabs/PHP_CodeSniffer/issues/366#issuecomment-66937317

> I'm not sure what PSR2 actually intends in this case. They don't mention the alternate syntax anywhere. So I still need to enforce the spaces after opening parentheses error, which means you code will look like this after I fix these issues: